lib/hammer_cli_foreman/user_mail_notification.rb

```diff
@@ -0,0 +1,54 @@
+module HammerCLIForeman
+
+  class UserMailNotification < HammerCLIForeman::Command
+    resource :mail_notifications
+    command_name 'mail-notification'
+
+    class AddMailNotificationCommand < HammerCLIForeman::Command
+      action :create
+      command_name 'add'
+
+      success_message _('User mail notifications was added.')
+      failure_message _('Could not add user mail notification')
+
+      build_options
+    end
+
+    class UpdateMailNotificationCommand < HammerCLIForeman::Command
+      action :update
+      command_name 'update'
+
+      success_message _('User mail notifications was updated.')
+      failure_message _('Could not update user mail notification')
+
+      build_options
+    end
+
+    class RemoveMailNotificationCommand < HammerCLIForeman::Command
+      action :destroy
+      command_name 'remove'
+
+      success_message _('User mail notification was removed.')
+      failure_message _('Could not remove user mail notification')
+
+      build_options
+    end
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      action :user_mail_notifications
+
+      output do
+        field :mail_notification_id, _('Id')
+        field :name, _('Name')
+        field :description, _('Description')
+        field :interval, _('Interval')
+        field :mail_query, _('Mail query') unless :mail_query.nil?
```

Comment:
What exactly you want to achieve here?

`:mail_query.nil?` is always `false`.

Reply:
Only with one type of mail notification, there is a mail query field so I thought this way it will show the field only when it is relevant. In a second check, the column will appear all the time, if there is a mail query it will appear in the column, if not it will be empty.
long story short - I removed the unless part it was redundant.